CHANGELOG.unreleased.md

```diff
@@ -12,8 +12,9 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 
 ### Added
 - Added the option to move a bounding box via dragging while pressing ctrl / meta. [#7892](https://github.com/scalableminds/webknossos/pull/7892)
+- Added the option to add metadata entries to dataasets and folders. The metadata can be viewed and edited in the dashboard in the right details tab.[#7886](https://github.com/scalableminds/webknossos/pull/7886)
```

Comment:
Maybe rename the PR to also use the word “Metadata” instead of Properties? (Or update the code to also use Properties) I’d like to introduce only one new term, not two :sweat_smile: